tartufo/cli.py

```diff
@@ -81,6 +81,13 @@ def get_command(self, ctx: click.Context, cmd_name: str) -> Optional[click.Comma
     show_default=True,
     help="Enable high signal regexes checks.",
 )
+@click.option(
+    "--scan-filenames/--no-scan-filenames",
+    is_flag=True,
+    default=True,
+    show_default="--scan-filenames",
+    help="Enable filename checks.",
```

Comment:
```suggestion
    help="Check the names of files being scanned as well as their contents.",
```
This could probably use a bit more word-smithing. But we should try to be more clear with our help text. Just saying "filename checks" isn't very clear to a user who isn't familiar with the tool.